tests/unit/template_stargate_test.go

```diff
@@ -105,7 +105,7 @@ var _ = Describe("Verify Stargate template", func() {
 		})
 
 		It("changing cluster name", func() {
-			clusterName := Sprintf("k8ssandra-clustername-%s", UniqueIdSuffix)
+			clusterName := Sprintf("k8ssandracluster%s", UniqueIdSuffix)
```

Comment:
As it turns out, the hyphens make it harder to diagnose failures related to name mismatches.